packages/design-picker/src/utils/designs.ts

```diff
@@ -24,8 +18,8 @@ export const getDesignPreviewUrl = (
 		pattern_ids: recipe?.pattern_ids?.join( ',' ),
 		vertical_id: options.verticalId,
 		language: options.language,
-		viewport_width,
-		viewport_height,
+		...( options.viewport_width && { viewport_width: options.viewport_width } ),
+		viewport_height: options.viewport_height || DEFAULT_VIEWPORT_HEIGHT,
```

Comment:
Is there any concern that we only set the default value to `viewport_height` instead of both?

Reply:
Yes, since `viewport_width` is something we introduced for the generated design picker, I thought that it would be good to only add this parameter when explicitly set. This way, other components that use this function (e.g.: static design picker) won't be impacted by it. Let me know if that makes sense!

Reply:
Not sure if I understood correctly, but `viewport_height` was 700 before #64654, so components like the static design picker have been using it. Except that now we are changing the default value from 700 to 1040!

Reply:
Ooops...I misread something 🙈 Yup, it was 700 before https://github.com/Automattic/wp-calypso/pull/64654. Let's keep setting the default value to it